app/src/org/commcare/preferences/GlobalPrivilegesManager.java

```diff
@@ -76,4 +94,42 @@ private static String getPrivilegeDisplayName(String privilegeName) {
         }
     }
 
+    public static void storeRetrievedAvailableApps(Vector<AppAvailableForInstall> availableAppsRetrieved) {
```

Comment:
Merp yea you're probably right about the DB thing, not sure why that didn't occur to me. I should probably do that refactor now, I think it would be pretty messy to release it this way and then try to migrate to that later.